python/paddle/fluid/tests/unittests/test_initializer.py

```diff
@@ -41,6 +41,13 @@ def output_hist(out):
 
 
 class TestConstantInitializer(unittest.TestCase):
+    def test_calculate_gain(self):
```

Comment:
shall we add test case of relu and selu?

Reply:
done, thx